prova.js

```diff
@@ -0,0 +1,22 @@
+const fastify = require('./fastify')
```

Comment:
I think this file should be deleted before merge to master.

Reply:
Absolutely